packages/malloy/src/malloy.ts

```diff
@@ -1409,6 +1414,34 @@ export class Explore extends Entity {
   public get structDef(): StructDef {
     return this._structDef;
   }
+
+  public toJson(): SerializedExplore {
```

Comment:
In this codebase the existing methods are styled `toJSON()` and `fromJSON()`,  in alignment with the standard Javascript `toJSON()` method. This is counter to Google's style guide for acronyms but when in Rome...

Reply:
Kept the names as toJSON() and fromJSON(). The toJSON() method is implicitly called when serialize one Explore object.